lisa/tools/python.py

```diff
@@ -11,7 +11,7 @@
 if TYPE_CHECKING:
     from lisa.node import Node
 
-from lisa.operating_system import Posix
+from lisa.operating_system import Posix, BSD
```

Comment:
`from lisa.operating_system import BSD, Posix`

Reply:
Reordered the import statements.